source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-This guide shows you how to create an application that uses the Node.js
-driver to connect to a MongoDB Atlas cluster.
+*Estimated time to complete this guide: 35 minutes.*
 
-Check for Node and NPM
-----------------------
+This guide shows you how to create an application that uses the
+**Node.js driver** to connect to a **MongoDB Atlas cluster**.
 
-Ensure your system has Node version 8 or later and NPM (Node Package
-Manager) installed. Run the following commands to check that you meet
-these requirements:
+The Node.js driver is an interface through which you can connect to and
```

Comment:
"You can connect to and communicate with MongoDB instances using the Node.js interface."

Trying to frame this as outcome-oriented vs. feature-oriented ("do this thing" vs. "this is what this is").

Reply:
This is to address your request to include "What is the Node.js driver?" (without linking to another page) and "Where do I go if I'm not on the page I'm looking for". The prior sentence (opening sentence) mentions what it's used for. I'm not sure this needs to be either outcome-oriented or feature-oriented as it's a definition, similar to the next paragraph about what Atlas is.
The sentence you provided restates the first sentence, so I'm not sure how to use it. Do you have a different suggestion?

Reply:
Perhaps the order is what's confusing here. What if the first sentence on this page (the first paragraph) becomes the second or the last sentence of the second paragraph? 

Another suggestion is:
```
The Node.js driver is an interface through which you can connect to and communicate with MongoDB instances. This guide shows you how to create an application that uses the Node.js driver to connect to a MongoDB Atlas cluster.

If you prefer to connect to MongoDB using a different programming language, select from the ones on our drivers documentation page.
```

Reply:
The order I placed it in is:
-What is the purpose of the guide
-What is the Node.js driver
-Where should I go if I'm in the wrong place
-What is Atlas

I think the order you are suggesting is:
-What is the Node.js driver
-What is the purpose of the guide
-?
-?

Doesn't it make more sense to mention the purpose of the guide before defining the Node.js driver?

Reply:
Current order (taken from your comment):
-What is the purpose of the guide
-What is the Node.js driver
-Where should I go if I'm in the wrong place
-What is Atlas

Potential proposed order:
-What is the purpose of the guide
-Where should I go if I'm in the wrong place
-What is the Node.js driver
-What is Atlas


Thoughts? If you don't agree then we can leave it as-is.

Reply:
I think either way is ok with me. The ordering I used seeks to explain what the Node.js driver is immediately so the user can decide whether they're in the right place (focus on Jasmine persona), and your proposed ordering centralizes the definitions.

I am not certain what the best way and there might ultimately not be one. I'll try it your way and maybe one of your questions in future user testing or metrics analysis can be "were you able to identify that you're on the documentation page for the appropriate language driver?"

Reply:
Sounds good, thanks Chris.